examples/alpha/dsf_deployment/variables.tf

```diff
@@ -238,7 +238,7 @@ variable "database_cidr" {
 variable "dra_version" {
   description = "The DRA version to install"
```

Comment:
I suggest adding to the description - Supported versions are 4.11.0.10.0.7 and up. Similar to Sonar.